ucore/packages.json

```diff
@@ -26,6 +26,7 @@
                 "distrobox",
                 "duperemove",
                 "hdparm",
+                "intel-compute-runtime",
```

Comment:
do you think `intel-compute-runtime` should be in `ucore-minimal`?  It enables containerized workloads rather than contribute to host operations like disk/NAS/hardware enablement, etc.

And a `ucore-minimal:nvidia` build would have the nvidia container runtime.